### ISSUE TITLE
finish pcb design

### ISSUE THREAD

**Issue by leomcelroy** [hackclub/gamelab-beaker#2]
We want to order some new test board by the end of May 2022. 

These are the components we know are on the board.

- [Raspberry Pi Pico with integrated charging circuit](https://shop.pimoroni.com/products/pimoroni-pico-lipo?variant=39335427080275)
- LiPo battery
- [6 tactile buttons](https://www.adafruit.com/product/1119)
- [ST7735 TFT 1.8" Screen](https://www.adafruit.com/product/358)
- [Slide switch](https://www.adafruit.com/product/805)
- Some sort of speaker
  - [Magnetic Buzzer](https://www.adafruit.com/product/160)
  or
  - [Adafruit I2S 3W Class D Amplifier Breakout - MAX98357A](https://www.adafruit.com/product/3006)
  - [Mini Metal Speaker w/ Wires - 8 ohm 0.5W PID: 1890](https://www.adafruit.com/product/1890)
- 2 Leds
- Some capacitors for noise reduction

we also want to make the board extensible and should add some female header pins to unused pins on the pico, power, and ground

- add extensible female header pins

**Comment by leomcelroy** [comment]
I'd love some assistance finishing up the KiCad designs. Would @Hugoyhu or @bykevinyang be interested in helping out with this?

**Comment by Hugoyhu** [comment]
Yeah! I can use KiCAD. Do you want to finish the audio first, or just design the PCB with it?

Sent from my iPhone

On May 18, 2022, at 3:17 PM, leomcelroy ***@***.***> wrote:

﻿

I'd love some assistance finishing up the KiCad designs. Would @Hugoyhu<https://github.com/Hugoyhu> or @bykevinyang<https://github.com/bykevinyang> be interested in helping out with this?

**Comment by leomcelroy** [comment]
I would just design the PCB with it.

**Comment by Hugoyhu** [comment]
Roger. Design in with theoretical software support then

Sent from my iPhone

On May 18, 2022, at 6:55 PM, leomcelroy ***@***.***> wrote:

﻿

I would just design the PCB with it.
